http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -63,14 +63,129 @@ paths:
               schema:
                 $ref: "#/components/schemas/LeonardoVersion"
       security: []
+
+## Runtime paths ##
+
+  /api/runtimes:
+    get:
+      summary: List all active runtimes
+      description: List all active runtimes, optionally filtering on a set of labels
+      operationId: listRuntimes
+      tags:
+        - runtime
+      parameters:
+        - in: query
+          name: _labels
+          description: >
+            Optional label key-value pairs to filter results by. Example:
+            Querying by key1=val1,key2=val2
+            returns all clusters that contain the key1/val1 and key2/val2 labels (possibly among other labels).
+            Note: this string format is a workaround because Swagger doesn't support free-form
+            query string parameters. The recommended way to use this endpoint is to specify the
+            labels as top-level query string parameters. For instance: GET /api/clusters?key1=val1&key2=val2.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: includeDeleted
+          description: Optional filter that includes any runtimes with a Deleted status.
+          required: false
+          schema:
+            type: boolean
+            default: false
+      responses:
+        "200":
+          description: List of runtimes
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ListClusterResponse"
```

Comment:
I kinda wanna expose different objects just so we don't have to carry over fields that don't make sense....but we can do that in a follow up PR

Reply:
Yeah maybe do that in a separate PR.. because currently in Leo code we are using the same objects/serializers for both paths